pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -356,40 +386,39 @@ func hasSubject(subjects []rbacv1.Subject, x rbacv1.Subject) bool {
 	return false
 }
 
-func (r *rbac) ensureRoleBindings(ctx context.Context, sa *corev1.ServiceAccount) error {
+func (r *rbac) ensureClusterRoleBindings(ctx context.Context, sa *corev1.ServiceAccount) error {
 	logger := logging.FromContext(ctx)
 
-	logger.Info("finding role-binding edit")
+	logger.Info("finding cluster-role-binding edit")
 	rbacClient := r.kubeClientSet.RbacV1()
 
-	editRB, err := rbacClient.RoleBindings(sa.Namespace).Get(ctx, "edit", metav1.GetOptions{})
+	editCRB, err := rbacClient.ClusterRoleBindings().Get(ctx, "edit", metav1.GetOptions{})
 
 	if err == nil {
-		logger.Infof("found rolebinding %s/%s", editRB.Namespace, editRB.Name)
-		return r.updateRoleBinding(ctx, editRB, sa)
+		logger.Infof("found clusterrolebinding %s", editCRB.Name)
+		return r.updateClusterRoleBinding(ctx, editCRB, sa)
 	}
 
 	if errors.IsNotFound(err) {
-		return r.createRoleBinding(ctx, sa)
+		return r.createClusterRoleBinding(ctx, sa)
 	}
 
 	return err
 }
 
-func (r *rbac) createRoleBinding(ctx context.Context, sa *corev1.ServiceAccount) error {
+func (r *rbac) createClusterRoleBinding(ctx context.Context, sa *corev1.ServiceAccount) error {
 	logger := logging.FromContext(ctx)
 
-	logger.Info("create new rolebinding edit, in Namespace", sa.GetNamespace())
+	logger.Info("create new clusterrolebinding edit")
 	rbacClient := r.kubeClientSet.RbacV1()
 
 	logger.Info("finding clusterrole edit")
-	_, err := rbacClient.ClusterRoles().Get(ctx, "edit", metav1.GetOptions{})
-	if err != nil {
+	if _, err := rbacClient.ClusterRoles().Get(ctx, "edit", metav1.GetOptions{}); err != nil {
 		logger.Error(err, "getting clusterRole 'edit' failed")
 		return err
 	}
 
-	rb := &rbacv1.RoleBinding{
+	crb := &rbacv1.ClusterRoleBinding{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:            "edit",
```

Comment:
is there an already existing clusterRoleBinding with name `edit` by default on OpenShift.
If no, let us rename the ClusterroleBinding to something like `openshift-pipelines-edit`